package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@salesforce/eslint-plugin-aura",
-  "version": "2.1.0",
+  "version": "3.0.0-beta1",
```

```suggestion
  "version": "3.0.0-beta.1",
```
(not that it really matters...)